Unused imports in Gallery

After updating a feature branch CI is indicating that we have some unused imports in one of the new galleries experiments:
```
  ./gallery/experiments/experimental_abinitio_pipeline_10073_jsb.py:30:1: F401 'os' imported but unused
  ./gallery/experiments/experimental_abinitio_pipeline_10073_jsb.py:37:1: F401 'aspire.image.Image' imported but unused
```
Not sure how `tox` didn't catch that before. Either way, I'll put in a quick patch.

--- a/gallery/experiments/experimental_abinitio_pipeline_10073_jsb.py
+++ b/gallery/experiments/experimental_abinitio_pipeline_10073_jsb.py
@@ -27,14 +27,12 @@ https://www.ebi.ac.uk/empiar/EMPIAR-10073
 # Import packages that will be used throughout this experiment.
 
 import logging
-import os
 from pathlib import Path
 
 import numpy as np
 
 from aspire.abinitio import CLSync3N
 from aspire.denoising import LegacyClassAvgSource
-from aspire.image import Image
 from aspire.reconstruction import MeanEstimator
 from aspire.source import ArrayImageSource, OrientedSource, RelionSource
 from aspire.utils import fuzzy_mask

